rules/palettes.yaml

```diff
@@ -20,6 +20,30 @@
 	PaletteFromCurrentTileset@decoration:
 		Name: terraindecoration
 		ShadowIndex: 1
+	PaletteFromFile@apolblue:
+		Name: apolblue
+		Filename: apolblue.pal
+		Alpha: 0.8
```

Comment:
These values won't work. PaletteFromFile doesn't support alpha values.

You need to create a PaletteFromPaletteWithAlpha derivate if you'd really intend to use alpha. Pretty sure examples already exist in this file.